File: Java/Day03Assignment/src/com/sonata/Employee.java
package com.sonata;

public class Employee extends MyOwnException implements MyInterface{
	int  empId;
	String empName;
	double empSal;
	
	public Employee(int id, String name, double sal, String a) {
		super(a);
		this.empId = id;
		this.empName = name;
		this.empSal = sal;
		
	}
	
	public void display() {
		System.out.println("Employee id is : "+empId);
		System.out.println("Employee Name : "+empName);
		System.out.println("Employee Salary : "+empSal);
	}


	@Override
	public void saCal() throws MyOwnException 
	{
		
		double yearlySal = empSal * 12;
		
			if(yearlySal < 200000)
			{
				throw new MyOwnException(" lessthgan 1 lakh");
			}else 
			{
				System.out.println("gretherthan 1 lakh");
			}
	
	}
	public static void main(String[] args) throws MyOwnException 
	{
		Employee emp = new Employee(21266, "Srikanth", 30000 , "hai");
		emp.display();
		emp.saCal();
		
	}
	

}

File: TypeScript_jan10/Employee.ts
export class Employee{
    empID:number = 123;
    empName:string = "Srikanth";
    empSal:number = 2345

    constructor(i:number, n:string, s:number){
        this.empID=i;
        this.empName=n;
        this.empSal=s;
    }
     display(){
         console.log(this.empID);
         console.log(this.empName);
         console.log(this.empSal);

     }
}

 var e1 = new Employee(124,"kranthi",345);
 e1.display();